source/replication.txt

```diff
@@ -104,30 +104,43 @@ Automatic Failover
 ------------------
 
 When a primary does not communicate with the other members of the set
-for more than 10 seconds, an eligible secondary will hold an election
+for more than 10 seconds, an eligible secondary holds an election
```

Comment:
electionTimeoutMillis seconds, which happens to default to 10

Reply:
Done, with some tweaks